test/unit/validations/metadata-schema.spec.ts

```diff
@@ -81,4 +81,15 @@ describe('Metadata Schema', () => {
 
     expect(result.ok).toBeTruthy()
   })
+
+  it('when deploying a base wearable the validation pass', async () => {
```

Comment:
Perhaps it would be nice to add a couple of tests for incorrect metadata, to check that the user messages are clear.